01-Authorization-RS256/README.md

```diff
@@ -4,6 +4,38 @@ This sample demonstrates how to protect endpoints in a Go API by verifying an in
 
 ## Getting Started
 
+If you haven't already done so, [sign up](https://auth0.com/signup) for your free Auth0 account and create a new API client in the [dashboard](https://manage.auth0.com/).
+
+Clone the repo or download it from the Golang API quickstart page in Auth0's documentation.
+
+### Add Your Credentials
+
+Open up the `main.go` file and on lines 15-17 you will see variables for `JWKS_URI`, `AUTH0_API_ISSUER`, and `AUTH0_API_AUDIENCE`. Update these values with your credentials and save the file.
```

Comment:
We shouldn't name the line numbers here because they'll likely change if we ever modify something in the sample :)